bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/init/NodeInitializers.java

```diff
@@ -97,9 +99,9 @@
     {
         // Default values, one for each instrument type.
         final Map<Instrument, ISPNodeInitializer<ISPObservation, SPObservation>> m =
-                Arrays.asList(Instrument.values()).stream().collect(Collectors.toMap(
+                Arrays.stream(Instrument.values()).collect(Collectors.toMap(
```

Comment:
Looks like an improvement. I wonder why it isn't just `Instrument.values().stream(). ...`

Reply:
I think it's because `Instrument.values()` is a raw array, so it can't have any methods in it and needs to be wrapped in an `Array` object.

The elegance of Java.